boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+struct Imu;
+
+/** Creates Imu struct used for reading data from sensor
```

Comment:
nit; When you add a description for the javadoc make sure that it looks like this.

```
/**
/* Creates Imu struct ...
```

Reply:
ok

Reply:
Will we be generating documentation from these Javadocs eventually?